packages/editor/src/components/navigable-toolbar/index.js

```diff
@@ -61,6 +61,12 @@ class NavigableToolbar extends Component {
 		}
 	}
 
+	componentDidMount() {
+		if ( this.props.focusOnMount ) {
```

Comment:
If it's part of the public interface, we should include a note in the `CHANGELOG.md` file.

Ideally also in `README.md` for the component, but it doesn't exist (until #10699).

Reply:
I added a note to the CHANGELOG.md.

Regarding the README.md given that it is already being added in https://github.com/WordPress/gutenberg/pull/10699 my plan is to add a commit in https://github.com/WordPress/gutenberg/pull/10699 after merging this PR updating its description to include the new property (provided it is not merged before if that is the case I will add the commit here).